### PR TITLE
ctpdev: orbitid fix and scalers logs improvements

### PR DESCRIPTION
**Pull request by lietava**

### CODE REVIEW

File: DataFormats/Detectors/CTP/include/DataFormatsCTP/Configuration.h [hunk 1]
```diff
@@ -205,12 +205,11 @@ class CTPRunManager
   int saveRunScalersToCCDB(int i);
   int saveRunConfigToCCDB(CTPConfiguration* cfg, long timeStart);
   static CTPConfiguration getConfigFromCCDB(long timestamp, std::string run);
-  CTPRunScalers getScalersFromCCDB(long timestamp, std::string);
+  CTPRunScalers getScalersFromCCDB(long timestamp, std::string, bool* ok);
```
Comment (sawenzel) [inline DataFormats/Detectors/CTP/include/DataFormatsCTP/Configuration.h]: why a pointer? Consider using `bool& ok`.

Comment (lietava) [reply DataFormats/Detectors/CTP/include/DataFormatsCTP/Configuration.h]: accepted.

**Review by sawenzel — APPROVED** [review]

**Comment by lietava** [comment]
@sawenzel : the errors above are not related to CTP code , right ?

**Comment by lietava** [comment]
@sawenzel @shahor02 : please, merge.

File: DataFormats/Detectors/CTP/src/Configuration.cxx [hunk 2]
```diff
@@ -857,8 +863,10 @@ CTPRunScalers CTPRunManager::getScalersFromCCDB(long timestamp, std::string run)
   auto ctpscalers = mgr.getSpecific<CTPRunScalers>(mCCDBPathCTPScalers, timestamp, metadata);
   if (ctpscalers == nullptr) {
     LOG(info) << "CTPRunScalers not in database, timestamp:" << timestamp;
+    ok = 0;
   } else {
     // ctpscalers->printStream(std::cout);
+    ok = 1;
   }
   return *ctpscalers;
```
Comment (shahor02) [inline DataFormats/Detectors/CTP/src/Configuration.cxx:871]: do you really need a copy of the object?
Note that for the DPL workflows we are outphasing custom access to CCDB objects in favour of DPL fetcher.

Comment (lietava) [reply DataFormats/Detectors/CTP/src/Configuration.cxx:871]: This is utility function, it  is used only in macros, e.g.:
https://github.com/AliceO2Group/AliceO2/blob/e01f74f3624f955d44b4c6b189c448b9eb646b38/Detectors/CTP/macro/GetScalers.C#L45
never in workflow.

**Pull request merged**